String color
BlobInfo3D[] blobs
